src/vstart.sh

```diff
@@ -1642,9 +1642,8 @@ if [ $with_jaeger -eq 1 ]; then
   -p 16686:16686 \
   -p 14268:14268 \
   -p 14250:14250 \
-  jaegertracing/all-in-one:1.20
-fi
-
+  quay.io/jaegertracing/all-in-one
+  fi
```

Comment:
the commit message reads 

> n servers like sepia labs

what does this mean?

Reply:
@ideepika could you please revert this change? the indent of `fi` is wrong.

Reply:
@tchaikov thanks, fixed